apps/channels/utils.py

```diff
@@ -1,8 +1,103 @@
 from apps.channels.exceptions import ExperimentChannelException
-from apps.channels.models import ChannelPlatform
+from apps.channels.models import ChannelPlatform, ExperimentChannel
 from apps.experiments.models import Experiment
 
 
+def validate_embedded_widget_request(token: str, origin_domain: str, team) -> tuple[bool, ExperimentChannel]:
```

Comment:
This could just return the channel since the only time it returns `True` is when it also returns a channel